layouts/posts/single.html

```diff
@@ -60,6 +60,17 @@ <h2 class="single-subtitle">{{ . }}</h2>
                         <i class="far fa-eye fa-fw"></i>&nbsp;<span class=leancloud-visitors-count></span>&nbsp;{{ T "views" }}
                     </span>&nbsp;
                 {{- end -}}
+                {{- if $comment.enable | and $comment.waline.enable | and $comment.waline.visitor -}}
+                    <span id="{{ .RelPermalink }}" class="leancloud_visitors" data-flag-title="{{ .Title }}">
+                        <i class="far fa-eye fa-fw"></i>&nbsp;<span class=leancloud-visitors-count></span>&nbsp;{{ T "views" }}
+                    </span>&nbsp;
+					<a href="#comments" id="post-meta-vcount" title="{{ T `viewComments` }}">
```

Comment:
The indentation should be the same as is on the 65th line.

Reply:
May be done ?